srcpkgs/mftrace/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'mftrace'
+pkgname=mftrace
```

Comment:
Why add this package ? seems unrelated

Reply:
No Idea, what this is doing here. Maybe I made some mistakes when rebasing